community/content.md

```diff
@@ -147,11 +148,51 @@ book. Check out the {doc}`Jupyter Book getting started
 guide<jb:start/overview>` for more information about
 Jupyter Book.
 
-You can tell Jupyter Book to place links *directly to your 2i2c JupyterHub* on each
+You can tell Jupyter Book to place links _directly to your 2i2c JupyterHub_ on each
 page that is served from a notebook. To do so, follow the [launch buttons for
 JupyterHubs
 instructions](https://jupyterbook.org/interactive/launchbuttons.html#jupyterhub-buttons-for-your-pages).
 Make sure that you configure your `jupyterhub_url` to point to the URL of your
 2i2c JupyterHub (e.g., `https://<your-hub>.2i2c.cloud`).
 This will use automatically [create nbgitpuller links](content:nbgitpuller)
 for you.
+
+## Deploy documentation with Jupyter Book
+
+2i2c can share "community documentation" alongside your hub under a special `docs.<COMMUNITY>.2i2c.cloud` subdomain. This feature can be used to establish a home for the best practice, useful resources, and rich visualisations built by your community.
+
+:::{caution} Community Documentation is Experimental
+This feature is an experimental addition by 2i2c. It may change as we learn more about how communities are using it, and develop new features.
+:::
+
+Using a [2i2c-authored template][2i2c-org/community-docs-template], you can quickly get started by customising an existing bare-bones Jupyter Book. This template includes examples of:
+
+- A [landing page][guide-landing].
+- Authoring content that uses [a Python Jupyter kernel][guide-kernel].
+- Using rich [cross-referencing] features to create a knowledge base.
+
+Why not start by [adding your own glossary][guide-glossary], or [adding your own logo][guide-logo]? See [](#learn-authoring) for more tips about authoring content with Jupyter Book 2 and the MyST Engine.
+
+### Deploy your site with Jupyter Book
+
+The current iteration of this feature requires you to deploy a Jupyter Book 2 static site from a GitHub repository using GitHub Pages. The community representative should use the [2i2c-org/community-docs-template] template to [create their own repository][use-template], and provide the 2i2c team with _temporary_ owner access. We will then ensure that various configuration such as the custom domain and GitHub Actions are set up correctly.
+
+(learn-authoring)=
+
+### Learn about authoring with Jupyter Book and MyST Markdown
```

Comment:
Do we need to explicitly say "we are only responsible for setting this up, not maintaining it"?

Reply:
I don't think so. These instructions make it clear that the "community representative" are intended to make the content changes and only bring in 2i2c for configuration changes.